nomad/server.go

```diff
@@ -1067,6 +1067,7 @@ func (s *Server) setupRpcServer(server *rpc.Server, ctx *RPCContext) {
 	server.Register(s.staticEndpoints.ClientStats)
 	server.Register(s.staticEndpoints.ClientAllocations)
 	server.Register(s.staticEndpoints.FileSystem)
+	server.Register(s.staticEndpoints.Agent)
```

Comment:
I'm assuming that this wasn't needed before because monitor was a streaming endpoint that was reigstered above only?

Reply:
Correct! this is the Agent's first exported/non streaming endpoint